### PR TITLE
Ctrl-\ to Comment/Uncomment

### PR DESCRIPTION
**Pull request by joeky888**
Ideas from [here](http://stackoverflow.com/questions/1676632/whats-a-quick-way-to-comment-uncomment-lines-in-vim).

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -336,6 +383,11 @@ call CreateShortcut("f6",":call ToggleColorColumn()<CR>", "inv")
 
 " Ctrl O - Netrw (:Explore)
 call CreateShortcut("C-o",":call OpenNetrw()<CR>", "inv", "noTrailingIInInsert", "cmdInVisual")
+
+" Ctrl \ - Toggle comments
+call CreateShortcut("C-\\", ":call ToggleComments('.')<CR>", "in")
+vnoremap <silent> <C-\> <ESC>:call ToggleComments("")<CR>
```
Comment (sd65) [inline vimrc]: Why did you add this line ?

call CreateShortcut("C-\\", ":call ToggleComments('.')<CR>", "in**v**") does the same job. 
Is there another reason ?

Comment (sd65) [reply vimrc]: But first, thanks for improving MiniVim 👍

**Comment by joeky888** [comment]
By the way, do you like to toggle autocomplete from current file?

See [this post](https://github.com/sd65/MiniVim/issues/23#issuecomment-263184221), F10 to On/Off.

This is really useful for me.

**Comment by sd65** [comment]
I have to refactor a bit this PR and I will merge it then. Thanks !

**Comment by joeky888** [comment]
PS:
`set cursorline " Highligt the cursor line`
Should be
`set cursorline " Highlight the cursor line`

`set ttyfast " Faster refraw`
Should be
`set ttyfast " Faster redraw
`

**Comment by sd65** [comment]
Thanks again for this!

I modified just a tiny bit the code. Actually, I also fixed a bug where you coultn't uncomment a "comment leader" without space after.

Example in a vimrc file:
```
"
```

Notice the optional space in the regex now:
```
execute 'silent s/^' . b:comment_leader .'\( \)\?//g'
```

**Comment by joeky888** [comment]
Thanks! this bothers me sometimes.
